packages/perseus/src/widgets/__testdata__/interactive-graph.testdata.ts

```diff
@@ -769,7 +769,6 @@ export const polygonQuestionDefaultCorrect: PerseusRenderer = {
                         [2.5, 4],
                         [1.5, 2],
                     ],
-                    match: "congruent",
```

Comment:
The removal of this line is kind of a long story.

- I changed the RTL tests for interative graph to use `tab` to select the first movable element
- This meant that the test for polygon was selecting the entire polygon instead of the first point
- The polygon question was graded based on congruence
- That meant that moving the entire polygon, without changing its shape, resulted in a correct answer
- That caused the test for answering incorrectly to fail.

To test that incorrect answers are graded correctly, we make the grading more strict. When `match` is not present, the points must be exactly where the content author put them for the question to be marked correct.